prereise/gather/data/BA_County_map.json

```diff
@@ -3207,5 +3207,13 @@
         "Zapata__TX",
         "Zavala__TX"
     ],
-    "SCL_TPWR": ["Thurston__WA"]
-}
+    "SCL_TPWR": ["Thurston__WA"],
+    "ERCO-C": ["Brazoria__TX","Calhoun__TX","Chambers__TX","Fort Bend__TX","Galveston__TX","Harris__TX","Jackson__TX","Matagorda__TX","Montgomery__TX","Victoria__TX","Waller__TX","Wharton__TX"],
```

Comment:
Please wrap these long lines to match the rest of the file.

Reply:
Fixed.